content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+## Introduction
+
+The {{< gloss >}}external control plane{{< /gloss >}} deployment model enables mesh operators to install and manage mesh control planes on separate external clusters. This deployment model allows a clear separation between mesh operators and mesh admins. Istio mesh operators can now run Istio control planes for mesh admins while mesh admins can still control the configuration of the control plane without worrying about installing or managing the control plane.
+
+## Requirements
+
+### Cluster
+
+This guide requires that you have two Kubernetes clusters with any of the
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}. First cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster where it has Istio default profile installed in the <code>istio-system</code> namespace.   It also has external istiod installed in the <code>external-istiod</code> namespace.  The external istiod is exposed on the ingress gateway from the <code>istio-system</code> namespace.  The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} which also provides configuration for the external istiod.
```

Comment:
It's clearer to show this with a picture

Reply:
The purpose of the `external-istiod` namespace is unclear. I added my "best guess" below.
```suggestion
supported Kubernetes versions: {{< supported_kubernetes_versions >}}.

The first cluster is the {{< gloss >}}external control plane{{< /gloss >}} cluster with the Istio `default` profile installed in the `istio-system` namespace. 
An ingress gateway, which provides mesh sidecars access to the external istiod, is also installed on this cluster in the `external-istiod` namespace.

The second cluster is a {{< gloss >}}remote cluster{{< /gloss >}} hosting the mesh. Its Kubernetes API server also provides the configuration for the control plane (istiod) running in the external cluster.
```

Reply:
Thanks for the feedback!  I'll add a diagram for sure, once I get the tests automated.

The ingress gateway is installed in the istio-system namespace, from the istio default profile.  External-istiod namespace hosts the istiod in the first cluster.

Reply:
That doesn't seem to make much sense, since the default profile already has istiod. Maybe once you add a diagram it will make more sense, but either way, I think this warrants more explanation. The conceptual model is simple: one istiod in the external cluster with one ingressgateway to access it. Why there are 2 namespaces with 2 istiod's is counter intuitive, so that needs to be explained, and ideally illustrated in a diagram.

Reply:
Agreed it is confusing.  Basically the default profile has istiod and ingress gw... the sole purpose of that istiod is to manage the ingress gw.

Then external istiod (2nd istiod) is exposed on the ingress gw.

Reply:
So the actual control plane is the istiod installed in the `external-istiod` namespace. In that case, I would suggest putting more emphasis on it, and downplay the second Istio installed in the `istio-system'  namespace, since it's just an implementation detail. What profile is used for the `external-istiod` Istio, also `default`? You should mention that. The profile used for the other istiod is really not important.

Why is the Istio in the `istio-system` namespace using the `default` profile anyway. If its only purpose is to configure the ingress gateway, couldn't the minimal profile be used for it?

Reply:
Yes both are using default profiles.

min profile for istio-system ns could work however min profile doesn't contain the ingress gw.

Reply:
reworded a bit, hope it is clear now!